src/expand.rs

```diff
@@ -234,20 +233,17 @@ fn transform_block(
         return;
     }
 
-    let inner = Ident::new(&format!("__{}", sig.ident), sig.ident.span());
-    let args = sig
-        .decl
-        .inputs
-        .iter()
-        .enumerate()
-        .map(|(i, arg)| match arg {
-            FnArg::SelfRef(_) | FnArg::SelfValue(_) => quote!(self),
-            FnArg::Captured(ArgCaptured {
-                pat: Pat::Ident(PatIdent { ident, .. }),
-                ..
-            }) => quote!(#ident),
-            _ => positional_arg(i).into_token_stream(),
-        });
+    let inner = format_ident!("__{}", sig.ident, span = sig.ident.span());
```

Comment:
This can be written as

```rust
let inner = format_ident!("__{}", sig.ident);
```

because we'll propagate the  ident's span by default.

Reply:
That's good to know!